Tests/GRPCCodeGenTests/Internal/Renderer/TextBasedRendererTests.swift

```diff
@@ -820,12 +820,55 @@ final class Test_TextBasedRenderer: XCTestCase {
 
   func testStruct() throws {
     try _test(
-      .init(name: "Structy"),
+      StructDescription(name: "Structy"),
       renderedBy: { $0.renderStruct(_:) },
       rendersAs: #"""
         struct Structy {}
         """#
     )
+    try _test(
+      StructDescription(
+        name: "Structy",
+        conformances: ["Foo"]
+      ),
+      renderedBy: { $0.renderStruct(_:) },
+      rendersAs: #"""
+        struct Structy: Foo {}
+        """#
+    )
+    try _test(
+      StructDescription(
+        name: "Structy",
+        generics: [.member("T")]
+      ),
+      renderedBy: { $0.renderStruct(_:) },
+      rendersAs: #"""
+        struct Structy<T> {}
+        """#
+    )
+    try _test(
+      StructDescription(
+        name: "Structy",
+        generics: [.member("T")],
+        whereClause: WhereClause(requirements: [.conformance("T", "Foo")])
+      ),
+      renderedBy: { $0.renderStruct(_:) },
+      rendersAs: #"""
+        struct Structy<T> where T: Foo {}
+        """#
+    )
+    try _test(
+      StructDescription(
+        name: "Structy",
+        generics: [.member("T")],
+        conformances: ["Hashable"],
+        whereClause: WhereClause(requirements: [.conformance("T", "Foo")])
```

Comment:
I know we're not testing the `WhereClause`, but should we add a second requirement to make sure both get rendered appropriately?